utils/test/keycodes.js

```diff
@@ -2,170 +2,63 @@
 		} );
 	} );
 
-	describe( 'MacOS', () => {
-		it( 'should output MacOS Clover for PRIMARY key', () => {
-			const shortcut = macShortcut( PRIMARY );
-			expect( shortcut ).toEqual( '⌘' );
+	describe( 'primaryShift', () => {
+		it( 'should output Ctrl+Shift text on Windows', () => {
+			const shortcut = displayShortcut.primaryShift( 'M', isMacOSFalse );
+			expect( shortcut ).toEqual( 'Ctrl+Shift+M' );
 		} );
 
-		it( 'should output option symbol for ALT key', () => {
-			const shortcut = macShortcut( ALT );
-			expect( shortcut ).toEqual( '⌥option' );
+		it( 'should output shift+command symbols on MacOS', () => {
+			const shortcut = displayShortcut.primaryShift( 'M', isMacOSTrue );
+			expect( shortcut ).toEqual( '⇧shift+⌘M' );
 		} );
+	} );
 
-		it( 'should output control caret for META key', () => {
-			const shortcut = macShortcut( META );
-			expect( shortcut ).toEqual( '⌃control' );
+	describe( 'secondary', () => {
+		it( 'should output Ctrl+Shift+Alt text on Windows', () => {
+			const shortcut = displayShortcut.secondary( 'M', isMacOSFalse );
+			expect( shortcut ).toEqual( 'Ctrl+Shift+Alt+M' );
 		} );
 
-		it( 'should output Shift unicode for SHIFT key', () => {
-			const shortcut = macShortcut( SHIFT );
-			expect( shortcut ).toEqual( '⇧shift' );
+		it( 'should output shift+option+command symbols on MacOS', () => {
+			const shortcut = displayShortcut.secondary( 'M', isMacOSTrue );
+			expect( shortcut ).toEqual( '⇧shift+⌥option+⌘M' );
 		} );
+	} );
 
-		it( 'should strip + between command and single ending character', () => {
-			const simpleShortcut = macShortcut( `${ PRIMARY }+B` );
-			expect( simpleShortcut ).toEqual( '⌘B' );
-
-			const modifiedShortcut = macShortcut( `${ ALT }+${ PRIMARY }+B` );
-			expect( modifiedShortcut ).toEqual( '⌥option+⌘B' );
+	describe( 'access', () => {
+		it( 'should output Shift+Alt text on Windows', () => {
+			const shortcut = displayShortcut.access( 'm', isMacOSFalse );
+			expect( shortcut ).toEqual( 'Shift+Alt+M' );
 		} );
 
-		it( 'should not strip + between command and other character', () => {
-			const shortcut = macShortcut( `${ PRIMARY }+Space` );
-			expect( shortcut ).toEqual( '⌘+Space' );
+		it( 'should output control+option symbols on MacOS', () => {
+			const shortcut = displayShortcut.access( 'M', isMacOSTrue );
+			expect( shortcut ).toEqual( '⌃control+⌥option+M' );
 		} );
 	} );
 } );
 
-describe( 'accessShortcut', () => {
-	it( 'should uppercase character', () => {
-		const shortcut = accessShortcut( 'm', isMacOSFalse );
-		expect( shortcut ).toEqual( 'Shift+Alt+M' );
-	} );
-
-	it( 'should output Shift+Alt text on Windows', () => {
-		const shortcut = accessShortcut( 'M', isMacOSFalse );
-		expect( shortcut ).toEqual( 'Shift+Alt+M' );
-	} );
-
-	it( 'should output control+option symbols on MacOS', () => {
-		const shortcut = accessShortcut( 'M', isMacOSTrue );
-		expect( shortcut ).toEqual( '⌃control+⌥option+M' );
-	} );
-} );
-
-describe( 'accessKeyCode', () => {
```

Comment:
Looks like there are only tests for the `rawShortcut`s are gone. I guess the logic is really basic now so it matters less that there are no tests. 🤷‍♂️

Reply:
Ah, good catch, forgot to add tests for rawShortcode...